File: whanau-agents/src/aether_summit/agents/rangahau_hauora.py
from aether_summit.agents.base_agent import BaseAgent

class RangahauHauoraAgent(BaseAgent):
    def __init__(self):
        super().__init__(
            name="Rangahau Hauora",
            role_description="Health and medical evidence synthesis specialist. Reviews clinical evidence for preterm care and outputs whānau-centered health research."
        )

File: src/ai/agents/rangahau-hauora.ts
import { BaseAgent } from "./base";
import { AgentState } from "@/ai/types";

export class RangahauHauora extends BaseAgent {
 constructor() {
 super({
 name: "rangahau_hauora",
 description: "Medical research and evidence synthesis",
 systemPrompt: `You are Rangahau Hauora. You synthesise medical evidence related to preterm birth and twin pregnancies in New Zealand.`,
 });
 }

 getSystemPrompt(state: AgentState): string {
 return this.config.systemPrompt;
 }
}
